extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -116,7 +113,7 @@ public String getName() throws ParsingException {
         assertPageFetched();
         String title = null;
         try {
-            title = getVideoPrimaryInfoRenderer().getObject("title").getArray("runs").getObject(0).getString("text");
```

Comment:
Same as above: since you often use `getVideoPrimaryInfoRenderer()`, it would be a good idea to save it in a variable